app/components/pages/HCMonitorTablePage.jsx

```diff
@@ -50,11 +53,17 @@ class HCMonitorTablePage extends React.Component<Props> {
           :
             location.pathname === "/ui" ?
             // this should route to the ui table
+            <div>
+              <UploadBtn text="UI" fileUploaded={(file) => this.setState({uiUploaded: file})} />
```

Comment:
The reason is because empathizing with an end user, I don't want the menu bar items jumping around when I switch between them. I wouldn't recommend it if the menu weren't centered, and auto-centering

Reply:
Got it. Yes, is a bit disorienting.  I'll thing about an alt solution, but this works for now.  Thanks again for your edits and feedback!! :raised_hands: